packages/by-name/OVMF-TDX/0001-verify-Hobs-instead-of-measuring-them.patch

```diff
@@ -1,6 +1,6 @@
-From 649d2928a8f6552ba0644e6f9a7f719dfbdabe60 Mon Sep 17 00:00:00 2001
-From: Tom Dohrmann <erbse.13@gmx.de>
-Date: Mon, 23 Sep 2024 10:57:09 +0200
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Moritz Sanft <58110325+msanft@users.noreply.github.com>
```

Comment:
That's not supposed to happen, I believe.

Reply:
This is the result of recreation the patch in the form we advertise in `dev-docs/patches.md`

Reply:
`git am` should not replace the author :thinking:

Reply:
I had to `git apply --3way`, as `git am --3way` didn't work for some reason I don't remember anymore. Maybe this changed it, but I'll revert the author change manually for now.